### PR TITLE
DOCS DOCS DUCS DUCKS DUCKS DUCKS DUCKS

### PR DESCRIPTION
**Pull request by johntyree**
Each more beautiful than the last.

### CODE REVIEW

**Comment by johntyree** [comment]
These aren't finished, but they describe some of the highest level relationships.

There isn't much description about the operational behavior yet, i.e. how the `UndeterminedClausePolicy` actually works, how requirements get translated into rules, etc.

File: docs/source/glossary.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Glossary
+========
+
+  Repository
```
Comment (cournape) [inline docs/source/glossary.rst]: Requirement is missing

**Comment by cournape** [comment]
wait, are goose ducks ?

**Comment by johntyree** [comment]
Unfortunately, geese are not ducks.

File: docs/source/glossary.rst (hunk 1 above)
Comment (johntyree) [inline docs/source/glossary.rst]: Yes, this glossary is still very incomplete.

There are also no cross references or anything else. I need to figure out how ReST and sphinx work.

**Comment by johntyree** [comment]
Check that this works on readthedocs.

**Comment by johntyree** [comment]
Also add `Requirement` to the glossary.

**Comment by johntyree** [comment]
I've added the `Requirement` to the glossary and this should build correctly on readthedocs when we merge.

Merging now!

**Pull request merged**